src/js/components/scenarios.js

```diff
@@ -0,0 +1,30 @@
+import { newText, newMultimedia, newContainer } from './helpers';
+
+const createImages = (dataImages) => {
+  const arrayImages = dataImages.map((element, number) => {
+    const imagesScenarios = newMultimedia('img', element, ['scenarios__image']);
+    const textScenarios = newText('p', [''], ['scenarios__text']);
+    textScenarios.innerHTML = element.text;
+    const containerImages = newContainer(
+      'div',
+      [textScenarios, imagesScenarios],
+      ['scenarios__container-images', `scenarios__images--${number}`],
+    );
+    return containerImages;
+  });
+  return arrayImages;
+};
+
+const scenariosComp = (data) => {
+  const mainTitle = newText('h1', data.title, ['scenarios__title']);
```

Comment:
this is note and H1, remember the H1, are the main titles in the entire page, this title could be an h3.

Reply:
Done. Updated  the h3 instead the h1